ci/vagrant/node_setup_node.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eux
+
+dnf -y install vagrant-libvirt libvirt-client
+systemctl start libvirtd.socket
+mkdir -p "$WORKDIR"
```

Comment:
Shouldn't this script ensure that `$WORDIR` is defined or use something similar to https://github.com/daos-stack/daos/blob/3c8b2f99b9604a81c3a3f065f719c919e2899f71/ci/functional/test_main_node.sh#L5-L10 to ensure this command was successful?  Alternatively would a `mkdir` function in `ci/common.sh`  that verified the success of the command and provided extra data if something went wrong (e.g. `df`) make sense which could then be used here?

Reply:
The `u` in https://github.com/daos-stack/daos/blob/3c8b2f99b9604a81c3a3f065f719c919e2899f71/ci/vagrant/node_setup_node.sh#L3
will ensure `$WORKDIR` as a value and the `e` in the same line will ensure that the `mkdir` doesn't return a failure.  Or rather, exit the script with an error if it does.